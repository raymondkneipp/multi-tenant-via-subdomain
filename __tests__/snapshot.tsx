import { render } from "@testing-library/react";
import Home from "@/pages/index";

it("renders homepage unchanged", () => {
	const mockData = {
		key: "hello",
		owner: "world",
		bio: "keep me healthy",
		color: "green",
	};

	const { container } = render(<Home data={mockData} />);
	expect(container).toMatchSnapshot();
});
