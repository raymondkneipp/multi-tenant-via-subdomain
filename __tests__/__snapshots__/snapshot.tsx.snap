// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main>
    <h1>
      <em
        style="background: green; border-radius: 0.25rem; padding: 0.5rem;"
      >
        world
      </em>
       
      is the owner of
       
      <em
        style="background: green; border-radius: 0.25rem; padding: 0.5rem;"
      >
        hello
      </em>
    </h1>
    <p>
      Bio: 
      keep me healthy
    </p>
  </main>
</div>
`;
